scrapinghub/client.py

```diff
@@ -630,6 +632,8 @@ def update_tags(self, add=None, remove=None):
 
             >>> job.update_tags(add=['consumed'])
         """
+        if not (add or remove):
+            raise ValueError('Please provide tags to add or remove')
```

Comment:
Well, it makes sense, I don't mind to add it 👌  But I still don't see any harm to have client validation for it, there's no point to do update() call without data, do you think it's redundant?